saml2/saml2-service-provider/src/test/java/org/springframework/security/saml2/provider/service/web/DefaultSaml2AuthenticationRequestContextResolverTests.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.security.saml2.provider.service.web;
+
+import org.junit.Test;
+import org.springframework.security.saml2.provider.service.authentication.Saml2AuthenticationRequestContext;
+
+public class DefaultSaml2AuthenticationRequestContextResolverTests {
+
+	private Saml2AuthenticationRequestContextResolver authenticationRequestContextResolver = new DefaultSaml2AuthenticationRequestContextResolver();
+
+	@Test(expected = IllegalArgumentException.class)
+	public void resolveWhenRequestAndRelyingPartyNullThenException() {
```

Comment:
Just a reminder to add tests that confirm common use cases.

Reply:
I added this test to cover the minimum test that needs to be done on this.

Reply:
Oh, sorry, what I meant was, remember to _also_ add tests that confirm to common use cases. Your test here is also appropriate.